<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>RzxChatPropUI</class>
 <widget class="QWidget" name="RzxChatPropUI" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>8</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item rowspan="2" row="1" column="0" >
    <widget class="QGroupBox" name="groupChat" >
     <property name="title" >
      <string>Chat</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>8</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="1" column="0" >
       <widget class="QCheckBox" name="cbPrintTime" >
        <property name="text" >
         <string>Print hh:mm:ss in chat</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QCheckBox" name="cbPropertiesWarning" >
        <property name="whatsThis" >
         <string>&lt;h3>warning&lt;/h3>&lt;p>If you check this you will receive an notification each time somebody check you properties&lt;/p>&lt;p>&lt;b>Warning :&lt;/b> in some cases, this option can be used to spam you with notification windows&lt;/p></string>
        </property>
        <property name="text" >
         <string>Warn me when properties are being checked</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" colspan="2" >
    <widget class="QGroupBox" name="groupBeep" >
     <property name="title" >
      <string>Beep</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>8</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="0" column="0" >
       <widget class="QCheckBox" name="chkBeep" >
        <property name="whatsThis" >
         <string>&lt;h3>Beep on message&lt;/h3>&lt;p>Check this if you want qRezix to play a sound when you receive en message&lt;/p></string>
        </property>
        <property name="text" >
         <string>Beep on message reception</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>2</number>
        </property>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Preferred</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>90</width>
            <height>10</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLineEdit" name="beepSound" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="whatsThis" >
           <string>&lt;h3>Beep on message&lt;/h3>&lt;p>Select the sound to play when you receive a new message&lt;/p></string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btnBeepBrowse" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="text" >
           <string>...</string>
          </property>
          <property name="autoRaise" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QGroupBox" name="groupNetwork" >
     <property name="title" >
      <string>Network</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>8</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="0" column="1" >
       <widget class="QSpinBox" name="chat_port" >
        <property name="toolTip" >
         <string>&lt;h3>Peer2peer port&lt;/h3>&lt;p>Define the port of the chat&lt;/p>&lt;p>You shouldn't have to modify this value&lt;/p></string>
        </property>
        <property name="maximum" >
         <number>32768</number>
        </property>
        <property name="value" >
         <number>5050</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QLabel" name="lblServerport" >
        <property name="text" >
         <string>Port:</string>
        </property>
        <property name="textFormat" >
         <enum>Qt::PlainText</enum>
        </property>
        <property name="buddy" >
         <cstring>chat_port</cstring>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>221</width>
       <height>198</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Smileys</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>8</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="0" column="0" >
       <widget class="QComboBox" name="smileyCombo" />
      </item>
     </layout>
    </widget>
   </item>
   <item rowspan="3" row="2" column="1" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>221</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <tabstops>
  <tabstop>cbPropertiesWarning</tabstop>
  <tabstop>cbPrintTime</tabstop>
  <tabstop>chkBeep</tabstop>
  <tabstop>beepSound</tabstop>
  <tabstop>btnBeepBrowse</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>chkBeep</sender>
   <signal>toggled(bool)</signal>
   <receiver>beepSound</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>150</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel" >
     <x>339</x>
     <y>93</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkBeep</sender>
   <signal>toggled(bool)</signal>
   <receiver>btnBeepBrowse</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>317</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel" >
     <x>588</x>
     <y>94</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
